Data-base/HOME WORK WEEK 1.sql

```diff
@@ -0,0 +1,51 @@
+-- Active: 1724582665781@@127.0.0.1@3308@hyf_lesson1
+Select COUNT(*) as total_tasks from task ;
+/*Find out how many tasks are in the task table*/
+
+SELECT COUNT(*) AS invalid_due_date_count FROM task WHERE due_date IS NULL;OR due_date = '';
```

Comment:
this query is not correct or at least incomplete, after you have the ; the OR will not be performed

your query will be

`SELECT COUNT(*) AS invalid_due_date_count 
FROM task 
WHERE due_date IS NULL;
`

instead of

`SELECT COUNT(*) AS invalid_due_date_count 
FROM task 
WHERE due_date IS NULL OR due_date = '';
`

Reply:
Can you check again i did it changes